.travis.yml

```diff
@@ -6,28 +6,34 @@ cache:
         - extra/*/vendor
         - $HOME/.composer/cache/files
 
+env:
+    global:
+        - SYMFONY_PHPUNIT_REMOVE_RETURN_TYPEHINT=1
+        - SYMFONY_PHPUNIT_DISABLE_RESULT_CACHE=1
+
 before_install:
     - phpenv config-rm xdebug.ini || return 0
 
 install:
     - travis_retry composer install
-    - (cd extra/cssinliner-extra && travis_retry composer install)
-    - (cd extra/html-extra && travis_retry composer install)
-    - (cd extra/inky-extra && travis_retry composer install)
-    - (cd extra/intl-extra && travis_retry composer install)
-    - (cd extra/markdown-extra && travis_retry composer install)
+    - ([[ $TRAVIS_PHP_VERSION = 7.0 ]] || cd extra/cssinliner-extra && travis_retry composer install)
+    - ([[ $TRAVIS_PHP_VERSION = 7.0 ]] || cd extra/html-extra && travis_retry composer install)
+    - ([[ $TRAVIS_PHP_VERSION = 7.0 ]] || cd extra/inky-extra && travis_retry composer install)
+    - ([[ $TRAVIS_PHP_VERSION = 7.0 ]] || cd extra/intl-extra && travis_retry composer install)
+    - ([[ $TRAVIS_PHP_VERSION = 7.0 ]] || cd extra/markdown-extra && travis_retry composer install)
```

Comment:
it might be more readable to override the `install` and `script` sections in the 7.0 job instead.